lib/vue/src/components/input/CInputConfirm.vue

```diff
@@ -13,13 +13,26 @@
         :class="`${buttonClass} ${borderless ? 'border-0' : ''}`"
         @click.stop.prevent="onPrompt"
       >
-        <slot>
+        <b-spinner
+          v-if="processing"
+          data-test-id="spinner"
+          class="align-middle"
+          small
+        />
+        <slot v-else>
           <font-awesome-icon
-            :icon="['far', 'trash-alt']"
+            v-if="isIconVisible"
```

Comment:
probably makes sense to call this hideIcon, since it should be present by default.
Either that or make this true by default, and remove the prop where its not needed/add it where it is.